ocs_ci/ocs/ui/page_objects/s3_vector_tab.py

```
@@ -0,0 +1,429 @@
+            self.do_send_keys(
+                self.s3_vector_loc["obc_bucketclass_input"], bucketclass_name
+            )
+        except (NoSuchElementException, TimeoutException):
+            try:
+                self.do_send_keys(
+                    self.obc_loc["bucketclass_text_field"], bucketclass_name
+                )
+            except (NoSuchElementException, TimeoutException):
+                raise TimeoutException(
+                    f"Bucket class search input not found for '{bucketclass_name}'"
+                )
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)